sqlite/test/general/stream.test.js

```diff
@@ -84,208 +77,4 @@ describe('STREAM', () => {
-          const stream = await STREAM.from(Images)
-          const result = await streamJson(stream)
-          expect(result.length).toEqual(3)
-        })
-      })
-
-      test('READ stream dataset from entity with LargeBinary columns', async () => {
-        const { Images } = cds.entities('test')
-        await cds.tx(async () => {
-          const select = SELECT(['ID', 'data']).from(Images)
-          const stream = await STREAM(['ID', 'data']).from(select)
-          const result = await streamJson(stream)
-
-          expect(result.length).toEqual(3)
-
-          // Decode LargeBinary columns from base64
-          expect(Buffer.from(result[0].data, 'base64').length).toEqual(7891)
-          expect(result[1].data).toEqual(null)
-          expect(Buffer.from(result[2].data, 'base64').length).toEqual(7891)
-        })
-      })
-    })
-
-    describe('WRITE', () => {
-      test('WRITE with incorrect data type results in error', async () => {
-        const { Images } = cds.entities('test')
-
-        const stream = new Readable({
-          read() {
-            // Push invalid data onto stream to simulate an error
-            this.push(1)
-          },
-        })
-        try {
-          await STREAM.into(Images).column('data').data(stream).where({ ID: 1 })
-        } catch (err) {
-          expect(err.code).toEqual('ERR_INVALID_ARG_TYPE')
-        }
-      })
-
-      test('WRITE stream property with .column and .where', async () => {
-        const { Images } = cds.entities('test')
-        const stream = fs.createReadStream(path.join(__dirname, 'samples/test.jpg'))
-
-        const changes = await STREAM.into(Images).column('data').data(stream).where({ ID: 1 })
-        expect(changes).toEqual(1)
-        await readStream(1)
-      })
-
-      test('WRITE stream property with keys as object in .into', async () => {
-        const { Images } = cds.entities('test')
-        const stream = fs.createReadStream(path.join(__dirname, 'samples/test.jpg'))
-
-        const changes = await STREAM.into(Images, { ID: 1 }).column('data').data(stream)
-        expect(changes).toEqual(1)
-        await readStream(1)
-      })
-
-      test('WRITE stream property with keys as integer in .into', async () => {
-        const { Images } = cds.entities('test')
-        const stream = fs.createReadStream(path.join(__dirname, 'samples/test.jpg'))
-
-        const changes = await STREAM.into(Images, 1).column('data').data(stream)
-        expect(changes).toEqual(1)
-        await readStream(1)
-      })
-
-      test('WRITE stream property on view', async () => {
-        const { ImagesView } = cds.entities('test')
-        const stream = fs.createReadStream(path.join(__dirname, 'samples/test.jpg'))
-
-        const changes = await STREAM.into(ImagesView, 1).column('renamedData').data(stream)
-        expect(changes).toEqual(1)
-        await readStream(1, ImagesView)
-      })
-
-      test('WRITE dataset from json file stream', async () => {
```

Comment:
It would be nice to retain this functionality. Probably should be revisited after the clean up.

Reply:
tests are changed, the idea is left as a comment